frontend-react/src/components/Snitch/PrettyPrintedPayloadBody.tsx

```diff
@@ -14,18 +14,37 @@ function decodeBody(value: string): string {
 }
 
 function asJson(content: string): string {
-  return JSON.stringify(JSON.parse(decodeBody(content)), null, "  ");
+  return JSON.stringify(JSON.parse(content), null, "  ");
 }
 
 function asXml(content: string): string {
-  return xmlFormatter(decodeBody(content));
+  return xmlFormatter(content);
 }
 
 interface BodyProps {
   contentType: string;
   content: string;
 }
 
+function asWwwFormUrlencoded(content: string): JSX.Element {
+  return (
```

Comment:
Man kan bruke nettleseren sin egen funksjonalitet for å parse uri-en:
```
const urlParams = new URLSearchParams('abc=foo&def=%5Basf%5D&xyz=5');
const params = Object.fromEntries(urlParams); // {abc: "foo", def: "[asf]", xyz: "5"}
```

Evt. bruke `Object.entries` i stedet for `Object.fromEntries`.